### ISSUE TITLE
Many examples contain references to Monty Python

### ISSUE THREAD

**Issue by ninjaaron** [BenLauwens/ThinkJulia.jl#23]
This isn't a problem per se, but I was scanning the first couple chapters and noticed many references to Monty Python, presumably included from _Think Python_. Python has a culture of using Monty Python references in example code, a gimmick which is kind of cute in that context, but it's a little conspicuous in a book which ostensibly has nothing to do with Python.

I'm not overly bothered by it, and maybe it's not worth bringing up at all, but I'm reading the book now and I could keep some notes about where Monty Python references appear and maybe find some replacement material. Maybe Babbage quotes or other quips from CS luminaries, get some PR's together.

I just thought I'd bring it up. Feel free to close this issue and ignore if this is stupid.

**Comment by BenLauwens** [comment]
See other comment... ;)

**Comment by chrisvwx** [comment]
@ninjaaron  I'm helping with a Spanish translation of ThinkJulia; and want to be aware of the Monty Python references, since I guess it will make even less sense for Spanish speakers.  Doing a search in the ThinkJulia book, I only found five references to "Monty". 

Did you keep that list of references?

**Comment by ninjaaron** [comment]
@christianpeel It's not just "Monty". It's all the examples that have to do with sausage, eggs and spam, references unladen swallows, pining for the fjords, only a flesh wound, "Nee!", John Cleese, Eric Idle and things like that. By tradition, Python instructional material is chalked full of this kind of stuff.

However, I haven't haven't gone through most of the book yet myself, since @BenLauwens encouraged me to wait until he was finished preparing the book for publications before I start any major work on addressing some of the Python-centric tendencies in the material (also applicable to semantic differences in the language.) See #25 for that conversation. Sorry I can't help more!

**Comment by OldSubSailor** [comment]
The Root Cause of all this Monty Python reference stuff was pretty much pointed out in the book, if y'all had read the Acknowledgments, in which he states "**I really want to thank Allen for writing Think Python and allowing me to port his book to Julia. Your enthusiasm is contagious!**"  So a ton of the _Think Python_ book is actually  this book. 

Lauwens, Ben; Downey, Allen B.. Think Julia . O'Reilly Media. Kindle Edition.

**Comment by ninjaaron** [comment]
@OldSubSailor Yes, I realize why all of the Monty Python references are in the book. I didn't open this issue in search of a root cause, but rather a solution. My feeling at the time I made this issue (four years ago) was that one might want to translate not only the operations from Python to Julia, but also use cultural content that was less specific to the Python community.

In retrospect, I don't know how big of a deal this is. I'm still more concerned with issue #25---that some of the terminology is actually more reflective of Python semantics than Julia semantics. The references to Monty Python are just a symptom of the same overall problem: the port of this book still contains some Python-specific content which is irrelevant or inaccurate with regard to Julia.
